libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2403,6 +2426,58 @@ bool WaveTrack::GetOne(
    return result;
 }
 
+ChannelGroupSampleView
+WaveTrack::GetSampleView(double t0, double t1, bool mayThrow) const
+{
+   assert(IsLeader());
+   ChannelGroupSampleView result;
+   for (const auto& channel : Channels()) {
+      result.push_back(channel->GetSampleView(t0, t1, mayThrow));
+   }
+   return result;
+}
+
+ChannelSampleView
+WaveChannel::GetSampleView(double t0, double t1, bool mayThrow) const
```

Comment:
I hope that writing this commit was a good exercise in understanding the new channel and interval interfaces.